switch base of buildimage to ubuntu noble

ernstblechaPT opened the pull request:
Hoping to resolve https://github.com/eclipse-4diac/4diac-forte/issues/287 with this pull request:

The update sites for ubuntu lunar have been removed from the mirrors. Switch the base image to a long-term version of ubuntu to avoid similar problems in the near future

ernstblechaPT committed:
switch base of buildimage to ubuntu noble

The update sites for ubuntu lunar have been removed from the mirrors. Switch the base image to a long-term version of ubuntu to avoid similar problems in the near future

ernstblechaPT commented:
Also: should libpcap-dev be already installed in the forte-buildimage instead of via the ci-configuration? (see https://github.com/eclipse-4diac/4diac-forte/pull/288)

mx990 commented:
> Also: should libpcap-dev be already installed in the forte-buildimage instead of via the ci-configuration? (see [eclipse-4diac/4diac-forte#288](https://github.com/eclipse-4diac/4diac-forte/pull/288))

Yes, it should. Could you please add it?

ernstblechaPT committed:
install libpcap-dev

add libpcap-dev into fortebuildcontainer to avoid having to install this ever time the container is used
